SteamList/Views/GameDescriptionViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  GameDescriptionViewController.swift
+//  SteamList
+//
+//  Created by Macbook on 26/01/2023.
+//
+
+import UIKit
+
+class GameDescriptionViewController: UIViewController {
+   
+    private let dividerBar: UIView = {
+        var view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
+        
+        return view
+    }()
+    
+    private let descriptionLabel: UILabel = {
+        var label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.lineBreakMode = .byClipping
+        label.textAlignment = .justified
+        label.text = "Every day, millions of players worldwide enter the battle as one of over a hundred Dota Heroes in a 5v5 team clash. Dota is the deepest multi-player action RTS game ever made and there's always a new strategy or tactic to discover. It's completely free to play and always will be – start defending your ancient now."
+        label.textColor = .white
+        
+        return label
+    }()
+    
+    private let bottomImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "ravage")
+        imageView.contentMode = .scaleToFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    
+    var constraints = [NSLayoutConstraint]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        constructViewHierarchy()
+    }
+    
+    override func viewDidLayoutSubviews() {
```

Comment:
Same here, you could add costraints without waiting

Reply:
What would you suggest, where should I put my addConstraints method